vignettes/errors.Rmd

```diff
@@ -85,24 +85,6 @@ initial(a, b) <- 1
 initial(x = a) <- 1
 ```
 
-# `E1004`
```

Comment:
I guess we can use `E1004` for some new error now, since noone will be expecting these to be stable?

Reply:
Yes, I was wondering about that. It will look a little odd to have non-contiguous errors in the file. I suppose one nice way would be to extract the things I've added in error 1005 here, and detect the compare() <- error more explicitly...